### PR TITLE
TNS archival submission

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR enables TNS "archival" submission (those without upperlimits).

### CODE REVIEW

File: services/tns_queue/tns_queue.py [hunk 1]
```diff
@@ -41,7 +41,15 @@
...
 
 
-def tns_submission(obj_ids, tnsrobot_id, user_id, reporters="", parent_session=None):
+def tns_submission(
+    obj_ids,
+    tnsrobot_id,
+    user_id,
+    reporters="",
+    archival=False,
+    archivalComment="",
```
Comment (Theodlz) [inline services/tns_queue/tns_queue.py:50]: let's use snake_case for the arguments, to keep it python friendly.

File: services/tns_queue/tns_queue.py [hunk 2]
```diff
@@ -51,6 +59,10 @@ def tns_submission(obj_ids, tnsrobot_id, user_id, reporters="", parent_session=N
         SkyPortal ID of User posting to TNS
     reporters : str
         Reporters to appear on TNS submission.
+    archival : boolean
+        Reporting the source as an archival source (i.e. no upperlimit).
+    archivalComment : str
```
Comment (Theodlz) [inline services/tns_queue/tns_queue.py:64]: same thing here in the docstring

File: skyportal/handlers/api/tns.py [hunk 3]
```diff
@@ -382,6 +382,8 @@ def post(self, obj_id):
             data = self.get_json()
             tnsrobotID = data.get('tnsrobotID')
             reporters = data.get('reporters', '')
+            archival = data.get('archival', False)
+            archivalComment = data.get('archivalComment', '')
```
Comment (Theodlz) [inline skyportal/handlers/api/tns.py:386]: here I would do `archival_comment = data.get('archivalComment', '')` instead, to have that snake_case

**Review by Theodlz — APPROVED** [review]
One minor comment, but otherwise LGTM

File: services/tns_queue/tns_queue.py (hunk 1 above)
Comment (mcoughlin) [reply services/tns_queue/tns_queue.py:50]: yes

File: services/tns_queue/tns_queue.py (hunk 2 above)
Comment (mcoughlin) [reply services/tns_queue/tns_queue.py:64]: good.

**Pull request merged**